db/migrate/20170316122243_create_events.rb

```diff
@@ -0,0 +1,12 @@
+class CreateEvents < ActiveRecord::Migration
+  def change
+    create_table :events do |t|
```

Comment:
Does an event not belong to any other model? ( is it not associated with another model? ) 🤔